datadog_checks_base/tests/base/checks/openmetrics/test_v2/test_first_scrape_handler.py

```diff
@@ -10,7 +10,6 @@
 
 pytestmark = [
     requires_py3,
```

Comment:
Side note: why isn't the linter picking up that this can fit on one <120 character line?

Reply:
Nice catch, I'll clean that up manually.